dev/sg/internal/run/pid.go

```diff
@@ -127,3 +130,35 @@ func writePid() error {
 
 	return nil
 }
+
+type logFile struct {
+	w      io.Writer
+	closer func()
+	path   string
+}
+
+func createLogFile() (*logFile, error) {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
```suggestion
logDir := filepath.Join(homeDir, ".sourcegraph", "logs")
```
and then use `logDir` for the paths below?